src/cfn/maybeSignS3HttpUrl.ts

```diff
@@ -0,0 +1,50 @@
+import * as aws from 'aws-sdk'
+import * as url from 'url';
+
+export function parseS3HttpUrl(input: string) {
```

Comment:
```suggestion
function parseS3HttpUrl(input: string) {
```

This isn't used outside of the module.